### ISSUE TITLE
Lesscpy incorrectly parses nested @media

### ISSUE THREAD

**Issue by oliverpool** [patacrep/patanet#120]
Voir lesscpy/lesscpy/issues/65.
Du coup le rendu est parfois erroné lors de la compilation... (par exemple l'alternance des couleurs sur la liste des artistes)

**Comment by Luthaf** [comment]
Au besoin, on peut switcher sur un parseur less plus complet, s'il est plus stable. Il y en a un en Ruby, non ?

**Comment by oliverpool** [comment]
L'original est en JS (Node.js donc)

**Comment by Luthaf** [comment]
Je ne sais pas, j'ai d'assez mauvais _a priori_ sur Node.js en prod. J'ai l'impression que les versions ne sont pas très stables.

**Comment by oliverpool** [comment]
Sinon on peut utiliser un truc du style https://github.com/niyazpk/LessCSS.windows

**Comment by oliverpool** [comment]
le problème, c'est que cela nécéssite d'être visité avec un navigateur : on pourrait le rajouter comme Vue admin uniquement

**Comment by Luthaf** [comment]
Je ne suis pas vraiment convaincu =) Je vais regarder si je peut faire quelque chose à base de os.Popen.

**Comment by oliverpool** [comment]
En fait le dépôt git précédent est désormais implémenté dans Less : http://lesscss.org/usage/#programmatic-usage (`less.render(lessInput, options)`)

Mais en tout les cas ça nécéssite un navigateur (avec un moteur javascript) : je ne comprends pas trop ce que tu veux faire avec os.Popen...

**Comment by Luthaf** [comment]
Appeller `lessc` lors du `collectstatic`.

Sinon, j'ai trouvé ça : https://github.com/miracle2k/webassets

**Comment by oliverpool** [comment]
Nécéssite Node.js : http://webassets.readthedocs.org/en/latest/builtin_filters.html#filters-css-compilers

**Comment by Luthaf** [comment]
Non, la version Ruby 2 [utilise CommonJS](https://rubygems.org/gems/less) pour faire tourner le fichier less.js.

**Comment by Luthaf** [comment]
Et il y a ce projet qui semble [gérer less](http://marklmiddleton.com/2013/using-less-with-django-on-heroku/) aussi : https://github.com/django-compressor/django-compressor

**Comment by oliverpool** [comment]
(au temps pour moi, je parlais de WebAssets, qui nécéssite node.js)

Je te laisse voir ce qui s'intègre le mieux avec le collecstatic.

**Comment by oliverpool** [comment]
`fade` n'est pas non plus géré par lesscpy... (page 'Aperçu' des lettres)
